packages/keyring-eth-ledger-bridge/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [11.0.1]
+
+### Fixed
+
+- Fix use of `salt` value as a `string` for `signTypedData<V4>` ([#249](https://github.com/MetaMask/accounts/pull/249))
+- Use timeouts for Ledger iframe messages ([#271](https://github.com/MetaMask/accounts/pull/271))
```

Comment:
This one could have been a new feature, but we keep it as a "fixed" in a sense that it fixes the current usage of the Ledger bridge.